github-profiles/NicoleU-3996/index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Nicole Urrego</title>  
```

Comment:
Aqui también agregaría el charset (UTF-8) para no tener inconvenientes con ciertos caracteres

Reply:
Define los caracteres que interpretará el sitio.. Te dejo un sitio de referencia que lo explique mas detallado https://www.w3schools.com/html/html_charset.asp 
"UTF-8 character set, which covers almost all of the characters and symbols in the world!"

Reply:
Bueno, muchas gracias :)